GUI/panels/scroll_panels.py

```diff
@@ -846,10 +846,14 @@ def get(key):
         # Add plugin injected chunks
         allowed = ['ScrollGeometryPanel_After', 'ScrollGeometryPanel_Before']
         if isinstance(plugin_chunks,dict):
-            for key,chunk in plugin_chunks.iteritems():
-                if key in allowed:
-                    core += chunk
-                    
+            try:
+                for key,chunk in plugin_chunks.iteritems():
+                    if key in allowed:
+                        core += chunk
+            except:
+                for key,chunk in plugin_chunks.items():
+                    if key in allowed:
+                        core += chunk                        

```

Comment:
Only the second one is needed I believe

Reply:
Splash screen works fine; issues are on the output panels as the results are not visible for some (possibly encoding) issues

Reply:
![image](https://user-images.githubusercontent.com/1848006/79049403-7b0ce980-7bf1-11ea-9593-3427542db331.png)